userpanel/modules/helpdesk/style/bclean/templates/helpdesklist.html

```diff
@@ -1,18 +1,34 @@
 {box title="Request history"}
+
+<script>
+	$(function() {
+		$(".table-click").click(function() {
+			window.document.location = $(this).attr("data-href");
+		});
+	});
+</script>
+
+<style>
+	.table-click tr,td {
+		color: black;
```

Comment:
Jeszcze to mi się nie podoba w tym miejscu.
Ogólnie to możliwe, że obsługa `table-click` powinna _przelecieć_ do jakiegoś jquery.js a nazwy klas powinny być zmienione na lms-userpanel-table-click (żeby nie mieć przyszłości kolizji z np. klasami wprowadzanymi przez bootstrap).

Reply:
Dopóki nie ma zdefiniowanych globalniejszych klas proponuję zostawić ASIS w myśl zasady działa-nie-ruszaj.

Reply:
To może w ogóle to wywalić?

Reply:
Zrobione.